LiveSplit.AutoSplitters.xml

```diff
@@ -13351,6 +13351,16 @@
         <Type>Script</Type>
         <Description>Happenlance Developer AutoSplitter</Description>
         <Website>https://github.com/pmttavara/HappenlanceAutoSplitter</Website>
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Inscryption</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Andersama/LiveSplitters/main/Inscryption.asl</URL>
+        </URLS>
```

Comment:
and now there's a case-sensitive problem here. The tags are matched with case-sensitiveness. You'd need to change the to